packages/core/src/components/image-overlay/image-overlay.scss

```diff
@@ -88,3 +91,63 @@ button {
   top: units.$u2;
   right: units.$u2;
 }
+
+ins.editaction-add {
+  display: inline-block;
+  margin-bottom: units.$u1;
+  padding: units.$u1;
+  @include insert.element();
+  @include insert.root();
+
+  .wrapper {
+    outline: none;
+    padding: units.$u1;
+    @include insert.root();
+
+    .editaction-label {
+      line-height: 1.5;
+    }
+
+    .close {
+      top: units.$u6;
+    }
+  }
+
+  .open {
+    top: units.$u6;
+  }
+}
+
+del.editaction-remove {
+  display: inline-block;
+  margin-bottom: units.$u1;
+  padding: units.$u1;
+  @include delete.root();
+
+  .wrapper {
+    outline: none;
+    padding: units.$u1;
+    @include delete.root();
+
+    .titel:not([hidden]) {
```

Comment:
Elders heb je refactors van EN->NL gedaan, en hier introduceer je een nieuwe NL class 🤡 

En is deze specifieke selector niet overbodig? Als het element met de class titel het `hidden` attribuut heeft, dan zie je het toch helemaal niet meer? Ergo, is het niet duidelijker als de text-decoration altijd wordt gezet?

Reply:
Deze class heb ik niet geïntroduceerd; bestond al. Heb deze wel EN gemaakt. 

Tevens :not([hidden]) eraf gehaald en de rendering conditioneel gemaakt op this.titelSlot